app/assets/javascripts/discourse/app/templates/hashtag-autocomplete.hbr

```diff
@@ -1,9 +1,12 @@
 <div class='autocomplete hashtag-autocomplete'>
-  <ul>
-    {{#each options as |option|}}
-      <li class="hashtag-autocomplete__option">
-        <a class="hashtag-autocomplete__link" title="{{option.description}}" href>{{d-icon option.icon}}<span class="hashtag-autocomplete__text">{{option.text}}</span></a>
-      </li>
-    {{/each}}
-  </ul>
+  <div class="hashtag-autocomplete__fadeout">
+    <ul>
+      {{#each options as |option|}}
+        <li class="hashtag-autocomplete__option">
+          <a class="hashtag-autocomplete__link" title="{{option.description}}" href>{{d-icon option.icon}}<span class="hashtag-autocomplete__text">{{option.text}}{{#if option.secondary_text}}<span class="hashtag-autocomplete__meta-text">({{option.secondary_text}}){{/if}}</span></span>
```

Comment:
could benefit from better formatting here please :)

Reply:
I feel the same way.